samples/helloworld/enclave/CMakeLists.txt

```diff
@@ -4,7 +4,9 @@
 # Use the edger8r to generate C bindings from the EDL file.
 add_custom_command(OUTPUT helloworld_t.h helloworld_t.c helloworld_args.h
   DEPENDS ${CMAKE_SOURCE_DIR}/helloworld.edl
-  COMMAND openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/helloworld.edl)
+  COMMAND openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/helloworld.edl
+                                 --search-path ${OE_INCLUDEDIR}/openenclave/edl
+                                 --search-path ${OE_INCLUDEDIR}/openenclave/edl/sgx)
```

Comment:
Ah, this is what I was hoping for :smile: I couldn't figure out which samples are supposed to work for OP-TEE so I didn't add OP-TEE support for any of them. I can update this to work for both for the samples that should work on OP-TEE.

Reply:
I have not tried the samples on OP-TEE (though I should...). I would expect `helloworld` and `file-encryptor` to work, and the others not to since the infrastructure for them is not implemented on OP-TEE (attestation, sealing, and switchless).

The tests known to work are listed in [ARM CI PR](https://github.com/openenclave/openenclave/pull/2527) (still pending; I've some work to do on it). See the _Further Work_ section, too.

Reply:
From a code perspective I would expect them to work, but from a build/signing perspective, all the samples are signed with oesign using .conf files which is why I assumed they didn't work on OP-TEE

Reply:
Closing based discussion in other thread.